handlers/api/validator_deposits_v1.go

```diff
@@ -0,0 +1,121 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethpandaops/dora/db"
+	"github.com/ethpandaops/dora/dbtypes"
+	"github.com/ethpandaops/dora/services"
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+)
+
+type ApiValidatorDepositsResponseV1 struct {
+	Amount                uint64 `json:"amount"`
+	BlockNumber           uint64 `json:"block_number"`
+	BlockTS               uint64 `json:"block_ts"`
+	FromAddress           string `json:"from_address"`
+	MerkleTreeIndex       string `json:"merkletree_index"`
+	PublicKey             string `json:"publickey"`
+	Removed               bool   `json:"removed"`
+	Signature             string `json:"signature"`
+	TxHash                string `json:"tx_hash"`
+	TxIndex               uint64 `json:"tx_index"`
+	TxInput               string `json:"tx_input"`
+	ValidSignature        bool   `json:"valid_signature"`
+	WithdrawalCredentials string `json:"withdrawal_credentials"`
+}
+
+// ApiValidatorDepositsV1 godoc
+// @Summary Get validator execution layer deposits
+// @Description Get all eth1 deposits for up to 100 validators
+// @Tags Validators
+// @Produce  json
+// @Param  indexOrPubkey path string true "Up to 100 validator indicesOrPubkeys, comma separated"
+// @Success 200 {object} types.ApiResponse{data=[]types.ApiValidatorDepositsResponseV1}
+// @Failure 400 {object} types.ApiResponse
+// @Router /api/v1/validator/{indexOrPubkey}/deposits [get]
+func ApiValidatorDepositsV1(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	vars := mux.Vars(r)
+	var param string
+	if r.Method == http.MethodGet {
+		// Get the validators from the URL
+		param = vars["indexOrPubkey"]
+	} else {
```

Comment:
this codepath is never hit as the endpoint accepts GET requests only.
can probably remove the whole if else block and use `param := vars["indexOrPubkey"]`

Reply:
I've removed the else part.